### DISCUSSION TITLE
Re-implement "Show Tutorial" Option in Main Menu

### DISCUSSION THREAD

**Discussion by OpenRift412** [OpenRCT2/OpenRCT2#18488] [Feature requests]
I'm a bit new to RCT2 and was just doing a little bit of comparing of vanilla RCT2 and OpenRCT2 when I noticed that the "Show Tutorial" option doesn't seem to be present in the main menu for OpenRCT2. Is the tutorial menu available elsewhere in this version or has it been stripped out entirely? If it's the latter, is there a reason for that? 

I think removing or obstructing the option kind of makes the experience a bit more confusing for newer players, so if possible I'd highly recommend restoring the option on the main menu.

**Comment by X123M3-256** [comment]
The tutorial was removed very early on because the way it was originally implemented was hardcoded to the screen resolution, it would have need a complete rewrite to work with arbitrary screen resolutions like OpenRCT2 supports, and this was never done.

**Reply by OpenRift412** [reply]
Damn, that's weird... Well I hope we can see it make a return at some point.

**Reply by IntelOrca** [reply]
Also any changes to the interface would be an issue as it was a recorded mouse input at specific locations.
